tests/grid/grid_generator_hyper_cross.cc

```diff
@@ -0,0 +1,70 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2005 - 2014 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+// Test output for GridGenerator::simplex()
```

Comment:
I think my rule of thumb is that if a test reuses a significant amount of code from a previous test, then I leave in the previous years of the copyright note. Only if the test is pretty completely new do I remove previous years.

I don't think we want to be dogmatic about it, though :-)